criu/cr-service.c

```diff
@@ -701,6 +702,8 @@ static int setup_opts_from_req(int sk, CriuOpts *req)
 	if (req->lsm_profile) {
 		opts.lsm_supplied = true;
 		SET_CHAR_OPTS(lsm_profile, req->lsm_profile);
+		pr_debug("opts.lsm_supplied %d\n", opts.lsm_supplied);
```

Comment:
It seems like `opts.lsm_supplied` is always `true` here.

Reply:
I don't think this belongs in this PR at all. I have to remove this.

Reply:
removed